tests/foreman/api/test_errata.py

```diff
@@ -199,17 +199,21 @@ def test_positive_install_multiple_in_host(
             'activationkey-id': ak.id,
         }
     )
-    # Install katello-ca, register content-host, enable all repos:
-    rhel_contenthost.install_katello_ca(target_sat)
-    rhel_contenthost.register_contenthost(function_org.name, ak.name, function_lce.name)
+    rhel_contenthost.register(
+        org=function_org,
+        activation_keys=ak.name,
+        lifecycle_environment=function_lce,
+        target=target_sat,
+        force=True,
+        loc=None,
+    )
     assert rhel_contenthost.subscribed
     # Installing outdated custom packages:
     for package in constants.FAKE_9_YUM_OUTDATED_PACKAGES:
         rhel_contenthost.run(f'yum remove -y {str(package.split("-", 1)[0])}')
         assert rhel_contenthost.run(f'yum install -y {package}').status == 0
         assert rhel_contenthost.run(f'rpm -q {package}').status == 0
     rhel_contenthost.add_rex_key(satellite=target_sat)
-    rhel_contenthost.run(r'subscription-manager repos --enable \*')
```

Comment:
Custom repos will be disabled by default on host, so are you sure this step is not required?

Reply:
@vsedmik any thoughts? I think Jeremy recommended against this when using `.register()` ?

Reply:
Also if custom repos are disabled as the test exists now, how could it pass locally and prt?

Reply:
I think because of [this](https://github.com/SatelliteQE/robottelo/blob/757f2a696ccb17ebcc2a98e7e5b2f2e925334262/robottelo/host_helpers/cli_factory.py#L676), which is executed by `target_sat.cli_factory.setup_org_for_a_custom_repo(...)`

Reply:
Oh my bad, I missed we're handling this in that helper, thanks !